src/lib/stripe/StripeCardInput.tsx

```diff
@@ -123,7 +137,7 @@ export const StripeCardInput: FC<StripeCardInputProps> = ({
         </label>
       )}
       <CardElement id={label} options={CARD_ELEMENT_OPTIONS} onChange={handleChange} onBlur={handleBlur} />
-      <ErrorMessage className="lc-stripe-card-input-error" name={name} component="div" />
```

Comment:
why did you get rid of the `ErrorMessage` component? Does that not style it all for you or something?

Reply:
It doesn't look for errors set on the `status`.